contracts/src/v0.8/tests/VerifiableLoadBase.sol

```diff
@@ -55,15 +57,14 @@ abstract contract VerifiableLoadBase is ConfirmedOwner {
   // the following fields are immutable bc if they are adjusted, the existing upkeeps' delays will be stored in
   // different sizes of buckets. it's better to redeploy this contract with new values.
   uint16 public immutable BUCKET_SIZE = 100;
-  uint16 public immutable TIMESTAMP_INTERVAL = 3600;
 
   /**
    * @param registrarAddress a registrar address
    * @param useArb if this contract will use arbitrum block number
    */
   constructor(address registrarAddress, bool useArb) ConfirmedOwner(msg.sender) {
-    registrar = KeeperRegistrar2_0(registrarAddress);
-    (, , , address registryAddress, ) = registrar.getRegistrationConfig();
+    registrar = AutomationRegistrar2_1(registrarAddress);
```

Comment:
nitpick, but you can just make the constructor arg type `AutomationRegistrar2_1` instead of `address` and then avoid the typecast